src/styles/respec.css.js

```diff
@@ -240,7 +244,7 @@ aside.example .marker > a.self-link {
 
   /* Don't position self-link in headings off-screen */
   :is(h2, h3, h4, h5, h6) + a.self-link {
-    left: 95%;
+    left: 101%;
```

Comment:
We can maybe follow Bikeshed here and always keep the self-link left of heading? Might simplify things.